backend/api/Areas/Tools/Controllers/GeocoderController.cs

```diff
@@ -83,6 +83,44 @@ public async Task<IActionResult> FindPidsAsync(Guid siteId)
             var result = await _geocoderService.GetPids(siteId);
             return new JsonResult(_mapper.Map<Model.SitePidsResponseModel>(result));
         }
+
+        /// <summary>
+        /// Make a request to Data BC Geocoder for the property that is the closest to the given lat/lng point.
+        /// </summary>
+        /// <param name="point">The lat/lng of the desired property.</param>
+        /// <returns>A single address match.</returns>
+        [HttpGet("nearest")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Model.AddressModel), 200)]
+        [ProducesResponseType(typeof(Pims.Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "tools-geocoder" })]
+        [HasPermission(Permissions.PropertyEdit)]
+        public async Task<IActionResult> FindNearestAddressAsync(string point)
+        {
+            var parameters = this.Request.QueryString.ParseQueryString<NearestParameters>();
+            parameters.Point = point;
```

Comment:
I would have though line 100 would have set the `Point` as it would have been part of the query.